X/xrootdgo/build_tarballs.jl

```diff
@@ -1,7 +1,7 @@
 using BinaryBuilder
 
 name = "xrootdgo"
-version = v"0.30.1"
+version = v"0.30.1b"
```

Comment:
Pkg/the registry don't like pre-release strings, just stick to X.Y.Z as format
```suggestion
version = v"0.30.1"
```

Reply:
but the `_jll` already merged, can we re-use the same version number?

Reply:
Yes, we always bump the build number in any case.  Which is another thing which Pkg/the registry don't like, but better to stick to a single thing which they don't like :smile: